static/js/compose.js

```diff
@@ -713,13 +736,13 @@ exports.handle_keydown = function (event) {
             // where "url" should be automatically selected.
             // Position of cursor depends on whether browser supports exec
             // command or not. So set cursor position accrodingly.
-            if (range.length > 0) {
+            if (range.text.length > 0) {
                 if (document.queryCommandEnabled('insertText')) {
                     txt.selectionStart = position - 4;
                     txt.selectionEnd = position - 1;
                 } else {
-                    txt.selectionStart = position + range.length + 3;
-                    txt.selectionEnd = position + range.length + 6;
+                    txt.selectionStart = position + range.text.length + 3;
+                    txt.selectionEnd = position + range.text.length + 6;
```

Comment:
This `range.text` change seems unrelated to the rest; can you give detail on this?

Reply:
Yeah, this changes doesn't require now. Cause we are changing the `range` rather than `range.text`. So they both will be same ideally. Removed.